src/components/input/input.spec.jsx

```diff
@@ -6,11 +6,6 @@ function render(props) {
   return shallow(<Input {...props} id="hi" />);
 }
 
-test('rendering', () => {
-  const wrapper = render();
-  expect(wrapper.find('input').prop('id')).toEqual('hi');
-});
-
 test('passing props through', () => {
```

Comment:
This is actually _also_ covered by the snapshot if you think about it. Ima leave this so we can see what we had before...but after, I'd like to just completely delete this spec if y'all agree!